contracts/HATVault.sol

```diff
@@ -672,18 +674,37 @@ contract HATVault is ERC4626Upgradeable, OwnableUpgradeable, ReentrancyGuardUpgr
     }
 
     /**
-    * @notice Called by governance to set the vault HAT token bounty split upon
-    * an approval. Either sets it to the value passed, or to the special "null" vaule, 
+    * @notice Called by governance to set the percentage of each claim bounty that
+    * will be swapped for hats and sent to the governance.
+    * Either sets it to the value passed, or to the special "null" vaule, 
+    * making it always use the registry's default value.
+    * @param _bountyGovernanceHAT The HAT bounty for governance
+    */
+    function setBountyGovernanceHAT(uint256 _bountyGovernanceHAT) external onlyRegistryOwner {
+        if (_bountyGovernanceHAT != NULL_UINT) {
+            registry.validateHATSplit(_bountyGovernanceHAT, getBountyHackerHATVested());
```

Comment:
I think it is beter to have a setter in which they are set together, given that also the validation is separate.

so `setBountyHat(_bountyGoverannceHAt, _bountyHackerHATVested)`